lib/smart_answer/calculators/paternity_pay_calculator.rb

```diff
@@ -12,6 +12,10 @@ def pay_duration
       paternity_leave_duration == "one_week" ? 1 : 2
     end
 
+    def paternity_deadline
+      date_of_birth < due_date ? (due_date + 55).strftime("%d-%m-%Y") : (date_of_birth + 55).strftime("%d-%m-%Y")
```

Comment:
```suggestion
      ([date_of_birth, due_date].max + 55).strftime("%d-%m-%Y")
```

This feels slightly neater because it avoids multiple calls to `strftime`. Although the `[date_of_birth, due_date].max` is a little bit magic so might be worth extracting that out into a variable.

Reply:
Now refactored 👍